### ISSUE TITLE
Can't paste when accessibility mode is enabled

### ISSUE THREAD

**Issue by joaomoreno** [microsoft/vscode#239319]
1. Have text in the clipboard
2. `Toggle Screen Reader Accessibility Mode`
3. Paste in an editor

🐛 Paste doesn't work

**Comment by Tyriar** [comment]
☝ was on macOS, but I see it on Windows too

**Comment by davidacm** [comment]
I'm facing this issue too since the last insider update. If I disable the screen reader support, the paste command works properly.

I tried the following without success:

* restart vscode with all extensions disabled (Developer: Reload With Extensions Disabled).
* call the paste command from the palete using f1 or ctrl + shift + p.

This command works in the file explorer, the console, and other text fields, but not in text editor.

**Comment by aiday-mar** [comment]
EditContext is turned on by default only on Insiders, so this does not need to be a candidate

**Comment by aiday-mar** [comment]
The fix is already in main, but reopening because this will be a candidate. The workaround is to disable the setting `editor.experimentalEditContextEnabled`

**Comment by osortega** [comment]
I can confirm that disabling the setting makes paste work

**Comment by tauhazmat** [comment]
Use Right-Click on mousepad or mouse. Simple as it is and try to use Command prompt instead of Power Shell.

**Comment by aiday-mar** [comment]
After talking in the standup, it was decided this issue would not be made into a candidate. So I removed the label and will be closing this issue.
